content/en/docs/collector/getting-started.md

```diff
@@ -106,7 +106,7 @@ Every Collector release includes DEB and RPM packaging for Linux amd64/arm64
 systems. The packaging includes a default configuration that can be found at
 `/etc/otel-collector/config.yaml` post-installation.
 
-> Please note that systemd is required for automatic service configuration
+> Please note that systemd is require for automatic service configuration
```

Comment:
```suggestion
> Please note that systemd is required for automatic service configuration
```

@austinlparker is this an automatic PR form the Collector repo? If so, ignore my comment and I'll fix the change there

Reply:
Yes, I think I triggered the automatic update.